### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by HasaluSL**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [ ] Resolved merge conflicts
- [ ] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -41,4 +41,8 @@
 - url: http://blog.kubacoding.eu/google-code-in/interview-with-prasenji-prasenji-gci-contestant-and-open-source-contributor/
   title: Interview with Prasenjit Gautam, GCI Contestant and Open Source Contributor
   img: 7.jpg
```
Comment (akshatnitd) [inline _data/blogs.yml:43]: Add a new line after this

File: _data/blogs.yml [hunk 2]
```diff
@@ -41,4 +41,8 @@
...
-  author_name: Kuba Michalski  
+  author_name: Kuba Michalski
```
Comment (akshatnitd) [inline _data/blogs.yml:44]: This should start with a ```-```

File: _data/blogs.yml [hunk 3]
```diff
@@ -41,4 +41,8 @@
 - url: http://blog.kubacoding.eu/google-code-in/interview-with-prasenji-prasenji-gci-contestant-and-open-source-contributor/
   title: Interview with Prasenjit Gautam, GCI Contestant and Open Source Contributor
   img: 7.jpg
-  author_name: Kuba Michalski  
+  author_name: Kuba Michalski 
+  
+- url: https://introductorymusicblockstutorial.wordpress.com/
```
Comment (akshatnitd) [inline _data/blogs.yml:46]: Remove ```-```

File: _data/blogs.yml [hunk 4]
```diff
@@ -41,4 +41,8 @@
...
+  title: Interview with Senel Chandrakeerthi, GCI Contestant and Fossasia user
+   author_name: Hasalu Botheju
```
Comment (akshatnitd) [inline _data/blogs.yml:48]: Correct the identation

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Check inline comments and also add an image of your blog

**Comment by the-codingninja** [comment]
Closing as you made another PR

**Pull request closed without merging**